src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py

```diff
@@ -24,6 +25,8 @@ def __init__(self, config):
         """
         BaseWorkerThread.__init__(self)
         self.config = config
+        # setup size threshold to fit CMSWEB nginx/frontend, i.e. 8MB
+        self.sizeThreshold = self.config.get('sizeThredhold', 8*1024*1024)
```

Comment:
this configuration has been placed at the top level configuration of the agent. Please add the component name to this (as you did for WorkflowUpdater).

Reply:
ok, done